src/jupyter_starters/schema/v2.json

```diff
@@ -146,6 +146,16 @@
           "description": "[SVG](https://developer.mozilla.org/en-US/docs/Web/SVG) string to use in Launcher cards and tab icons",
           "type": "string"
         },
+        "category": {
+          "title": "Category",
+          "description": "Category to place the starter card under in Launcher",
+          "type": "string"
+        },
+        "rank": {
+          "title": "Rank",
+          "description": "Rank of the launcher card within the category",
```

Comment:
as this drives the documentation, feel free to give a bit more information, e.g. _higher numbers will appear to the right and below lower numbers_ or whatever it actually does.

Reply:
also add a `default`

Reply:
See: https://github.com/deathbeds/jupyterlab-starters/pull/78

Added some more info, not sure if a default is really required. The other properties (icon, description, etc) also don't have a default.

But I added one for both now. The default of category is "Starters", however the default of rank is a bit arbitrary, since it is actually Infinity if not provided.

Please comment on the new PR to let me know what you think: https://github.com/deathbeds/jupyterlab-starters/pull/78